src/shared/components/entry/SequenceSection.tsx

```diff
@@ -1,17 +1,22 @@
 import { FC } from 'react';
 import { Card, Sequence } from 'franklin-sites';
 
-import { formatLargeNumber, hasContent } from '../../../shared/utils/utils';
+import { formatLargeNumber, hasContent } from '../../utils/utils';
 
 import EntrySection, {
   getEntrySectionNameAndId,
-} from '../../types/entrySection';
+} from '../../../uniref/types/entrySection';
 
-import { SequenceUIModel } from '../../../uniprotkb/adapters/sequenceConverter';
+export type Sequence = {
```

Comment:
Since it's used in other places maybe this should live in `shared/types` rather than within `shared/components`